src/main/java/org/dita/dost/util/DITAOTCopy.java

```diff
@@ -89,7 +89,11 @@ public void execute() throws BuildException {
             throw new BuildException("Destination directory not defined");
         }
         if (!destDir.exists()) {
-            throw new BuildException("Destination directory " + destDir + " does not exists");
+            try {
+                destDir.mkdir();
```

Comment:
`SecurityException` is not a check exception (i.e. it extends `RuntimeException`), thus we should not catch it here, just let cascade up.